docs/zh_cn/04-developer-guide/do_regression_test.md

```diff
@@ -49,56 +70,65 @@ python ./tools/regression_test.py \
 
 ## 例子
 
-1. 测试 mmdet 和 mmpose 的所有 backend 的 转换+精度
+1. 测试 mmdet 和 mmpose 的所有 backend 的 **转换+精度**
 
 ```shell
 python ./tools/regression_test.py \
-    --deploy-yml ./configs/mmdet/mmdet_regression_test.yaml ./configs/mmpose/mmpose_regression_test.yaml \
+    --codebase mmdet mmpose \
     --backends all \
```

Comment:
default of `backends` is None. Cannot pass `all` as backend for this version.

Reply:
Fixed